packages/firestore/src/api/database.ts

```diff
@@ -192,6 +192,37 @@ class FirestoreSettings {
       'timestampsInSnapshots',
       settings.timestampsInSnapshots
     );
+
```

Comment:
FYI- I moved these log messages from configureClient() to here because in configureClient(), the default has already been applied and so we can't differentiate if the setting is unset or set.